### PR TITLE
PD-300 Pool delete

### PR DESCRIPTION
**Pull request by nathan-px**
Anticipated feature release: 2.2.1
Draft due date: 
Spec review due date: 
Related JIRAs: PD-300
Stakeholders: @prabirpaul 
User stories: Ops staff

## Outline of Changes:

I've already added this to CLI section, but I'm not sure if we want to integrate it into instructions somewhere.

If we do, I'm not too sure where to put this. We don't have a storage pool operations, so it probably needs to be added as part of a customer use-case:

Maybe somewhere such as: https://docs.portworx.com/cloud-references/auto-disk-provisioning/vsphere/operations-guide/#installation
or 
https://docs.portworx.com/portworx-install-with-kubernetes/storage-operations/

## Concept information

* Use the Portworx CLI to delete storage pools which may be misconfigured or otherwise not functioning properly. 
 
### Prerequisites

* Your target pool for deletion must be empty and contain no replicas
* If your target pool for deletion is a metadata pool, it must be readable
* You must have more pools on the node than just your target pool for deletion

1. Place Portworx into [maintenance mode](/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/maintenance-mode)
2. Enter the `pxctl service pool delete` command followed by the `<pool_id>` of the pool you want to delete:

```
pxctl service pool delete 3
```

## Reference information

CLI docs

@prabirpaul How does this information relate to pool _deletion_? 
* Pool IDs are assigned incrementally. 
* A newly added pool always uses ID == last pool_id + 1. 
* The last usable pools id is 32.

### CODE REVIEW

File: content/reference/CLI/service.md [hunk 1]
```diff
@@ -441,3 +441,22 @@ You can use the `pxct service pool cache command` command to:
...
+### pxctl service pool delete
+
+You can use the `pxctl service pool delete` command to delete storage pools which may be misconfigured or otherwise not functioning properly.
+
+Consider the following requirements:
+
+* Your target pool for deletion must be empty and contain no replicas
+* If your target pool for deletion is a metadata pool, it must be readable
+* You must have more pools on the node than just your target pool for deletion
+* You must place your cluster in maintenance mode to use this command
```
Comment (prabirpaul) [inline content/reference/CLI/service.md]: cluster -> node

**Comment by prabirpaul** [comment]
The reference info is regarding pool addition after a delete.

File: content/reference/CLI/service.md (hunk 1 above)
Comment (vilasd) [inline content/reference/CLI/service.md]: its not cluster , its node in maintenance mode

**Comment by nathan-px** [comment]
@prabirpaul @vilasd 
image

I tried to keep the formatting and brevity in line with what's already there.

**Pull request merged**